examples_tests/10.AllocatorTest/main.cpp

```diff
@@ -348,85 +348,126 @@ int main()
 		nbl::core::address_allocator_traits<core::GeneralpurposeAddressAllocatorMT<uint32_t, std::recursive_mutex> >::printDebugInfo();
 	}
 
-	// Alloc pref test
+	constexpr uint32_t WIN_W = 1280;
+	constexpr uint32_t WIN_H = 720;
+	constexpr uint32_t SC_IMG_COUNT = 3u;
+
+	auto initOutp = CommonAPI::Init<WIN_W, WIN_H, SC_IMG_COUNT>(video::EAT_OPENGL, "Compute Shader");
```

Comment:
all of this is pointless, i dont think you need any GPU stuff in this test

Reply:
Yeah true, will  get rid of that